setcontext/main.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+import sys
+import pathlib
+
+#Add pytest
+#Build framework for  commandline tools.
+
+"""
+function to handle bad version string
```

Comment:
Use google style guide for docstrings. 
https://google.github.io/styleguide/pyguide.html#383-functions-and-methods. Function docstring should be after the function definition and there should be a brief one line docstring followed by more verbose docstring if necessary.

Reply:
IDEs like PyCharm have an auto complete for google style docstrings.